saucal/woocommerce-gateway-amazon-pay
# Add estimated order amount to pay button

### All Submissions:


### Changes proposed in this Pull Request:

Add estimatedOrderAmoun to Amazon Pay button.

### How to test the changes in this Pull Request:

1. Add a product to the cart.
2. Make sure that amazon pay button is active.
3. Try to update the cart items (change the quantity of the added item, add a new item or remove the existing one)
4. The following message is logged on browser's console (sandbox mode): `Estimated order amount for sandbox is {"amount":"XXX","currencyCode":"XXX"}`

### Other information:




### Changelog entry

> Add - Estimated order amount setting on pay button.
